src/component/loader/DropZone.tsx

```diff
@@ -58,9 +58,8 @@ function DropZone(props) {
         .then((fileCollection) => {
           dispatch({ type: LOAD_DROP_FILES, fileCollection });
         })
-        .catch((e: any) => {
+        .catch(() => {
           dispatch({ type: SET_LOADING_FLAG, isLoading: false });
-          reportError(e);
```

Comment:
Why did you remove this?

Reply:
Because we are not handle it at the level of nmrium, unless if the purpose of it to let others to listen for the error event outside nmrium.

Reply:
I added it for us developers. The default behavior is similar to `console.error(e)`

Reply:
ok, I will revert the changes